test/fixtures/plugin-express.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+var TraceLabels = require('../../src/trace-labels.js');
+var shimmer = require('shimmer');
+var semver = require('semver');
+var methods = require('methods').concat('use', 'route', 'param', 'all');
+
+var SUPPORTED_VERSIONS = '4.x';
+
+module.exports = function(version, api) {
+  if (!semver.satisfies(version, SUPPORTED_VERSIONS)) {
```

Comment:
Makes sense, it does seem boilerplate - I'm taking this to mean that plugins should be able to provide a field with that is passed to `semver.satisfies` internally.